tools/Makefile

```diff
@@ -48,3 +49,6 @@ $(1): $($1_SOURCES)
 endef
 
 $(foreach p,$(PROGRAMS),$(eval $(call COMPILE,$(p))))
+
+ido_recomp:
```

Comment:
Instead of a phony target, I would recommend a real target based on either the full directory or based on one of the executables. This way if ido_recomp already exists we don't spend time downloading it again.

Reply:
We would want to redownload sometimes even though ido_recomp exists (if we bump the ido-recomp version)?

Reply:
alright, I was reluctant because I really don't like making users do a manual step to update their repo after a change. But I guess it would be rare and there are already some changes that would require a `make distclean` to update anyway